versioned_docs/version-8.1/components/modeler/desktop-modeler/flags/flags.md

```diff
@@ -72,12 +72,14 @@ To display a custom version information in the status bar of the app, configure
 
 ### Zeebe SSL certificate
 
-To use a self-signed certificate to connect to Camunda Platform 8 Self-Managed, configure your `flags.json` like this:
+> ℹ️ The Modeler will read trusted certificates from your operating systems trust store.
+
+Provide an additional root certificate that the modeler should use to to validate secured connections to a Camunda Platform 8 Self-Managed installation. In case your server certificate is signed by a signing authority configure the CA certificate here, not the server certificate.
```

Comment:
```suggestion
Provide an additional root certificate that the modeler should use to validate secured connections to a Camunda Platform 8 Self-Managed installation. In case your server certificate is signed by a signing authority, configure the CA certificate here, not the server certificate.
```